### DISCUSSION TITLE
lessons/Syllabary-Dictation/day-3-pt2/

### DISCUSSION THREAD

**Discussion by giscus** [CherokeeLanguage/CherokeeLessons.com#15] [General]
# lessons/Syllabary-Dictation/day-3-pt2/

https://www.cherokeelessons.com/lessons/Syllabary-Dictation/day-3-pt2/

**Comment by MtRider94** [comment]
Osiyo Mr. Conrad,
I was wondering if there will be introduction videos for the remainder of the lessons?
I find that the introduction videos help me immensely in learning the shape recognition and flow of each shape.
Wado.

**Reply by michael-conrad** [reply]
Sorry I missed your comment, it didn't notify me. I'll check with the group and see if we can get any volunteers to finish those out.

**Reply by michael-conrad** [reply]
See: https://github.com/CherokeeLanguage/CherokeeLessons.com/issues/18
